clients/extension/manifest.json

```diff
@@ -5,24 +5,17 @@
   "manifest_version": 2,
   "description": "__MSG_appDescription__",
   "icons": {
-    "16": "images/lh_logo_bg.png",
-    "128": "images/lh_logo_bg.png"
+    "16": "images/lh_favicon_16x16.png",
+    "128": "images/lh_logo_128x128.png"
   },
   "default_locale": "en",
-  "background": {
-    "scripts": [
-      "scripts/lighthouse-ext-bundle.js"
-    ],
-    "persistent": false
-  },
   "permissions": [
     "activeTab",
-    "debugger",
     "storage"
   ],
   "browser_action": {
     "default_icon": {
-      "38": "images/lh_logo_icon.png"
```

Comment:
38 is a weird choice

Reply:
apparently it used to be 38 - I found old info. https://stackoverflow.com/questions/20424425/recommended-size-of-icon-for-google-chrome-extension

seems 32 is the thing now. https://developer.chrome.com/extensions/browserAction